### PR TITLE
NECO-113 renamed the library name.

### PR DESCRIPTION
**Pull request by sohda**
## Changes
- Renamed the library name.
### For the Source Code

Tested the sample project can run as Xcode workspace by select "RicohAPIMstorage.xcworkspace" file.
1. Run `git clone --recursive https://github.com/hishida/media-storage-swift.git -b feature/NECO-113`
2. Run `open media-storage-swift/RicohAPIMstorage.xcworkspace`
3. Choose `RicohAPIMstorageSample` scheme at the scheme menu of Xcode and run `RicohAPIMstorageSample`.
### For the Test Code

Tested the empty test code can run as Xcode workspace.
1. Run `git clone --recursive https://github.com/hishida/media-storage-swift.git -b feature/NECO-113`
2. Run `open media-storage-swift/RicohAPIMstorage.xcworkspace`
3. Choose `RicohAPIMstorage` scheme at the scheme menu of Xcode.
4. Click `MediaStorageTests.swift` 
5. Click the run button the left side of 9th line: `class MediaStorageTests: XCTestCase {`.

### CODE REVIEW

**Comment by hishida** [comment]
Now I'm reviewing...

File: RicohAPIMstorage.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:Sample/MediaStorageSample.xcodeproj">
+      location = "group:/Users/tomoko/SVaaS/Tickets/neco112/media-storage-swift/Sample/RicohAPIMstorageSample.xcodeproj">
    </FileRef>
    <FileRef
-      location = "group:MediaStorage.xcodeproj">
+      location = "group:/Users/tomoko/SVaaS/Tickets/neco112/media-storage-swift/RicohAPIMstorage.xcodeproj">
```
Comment (hishida) [inline RicohAPIMstorage.xcworkspace/contents.xcworkspacedata]: This line includes local absolute path, too. It should change to relative path.

File: RicohAPIMstorage.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -2,9 +2,9 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:Sample/MediaStorageSample.xcodeproj">
+      location = "group:/Users/tomoko/SVaaS/Tickets/neco112/media-storage-swift/Sample/RicohAPIMstorageSample.xcodeproj">
```
Comment (hishida) [inline RicohAPIMstorage.xcworkspace/contents.xcworkspacedata]: This line includes local absolute path. It should change to relative path.

File: RicohAPIMstorage.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (hishida) [inline RicohAPIMstorage.xcworkspace/contents.xcworkspacedata]: This line includes local absolute path, too. It should change to relative path.

File: RicohAPIMstorage.xcworkspace/contents.xcworkspacedata (hunk 2 above)
Comment (sohda) [inline RicohAPIMstorage.xcworkspace/contents.xcworkspacedata]: Thank you so much!
I'll fixed it right away.

File: RicohAPIMstorage.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (sohda) [inline RicohAPIMstorage.xcworkspace/contents.xcworkspacedata]: I'll fixed it right away.

**Comment by hishida** [comment]
I reviewed again and found no problems. I will merge this.

**Pull request merged**